etna/search/views.py

```diff
@@ -670,36 +669,6 @@ def add_article_page_for_url(self, page: Page) -> None:
                         f"WebsiteSearchView:scraped/ingested url={absolute_url} not found in wagtail_pages={wagtail_pages}"
                     )
 
-    def add_results_page_for_url(self, page: Page) -> None:
```

Comment:
@jamesbiggs We also need to make sure any code calling this method is updated :)